README.md

```diff
@@ -34,6 +34,8 @@ For `tap_action` options, see <https://www.home-assistant.io/dashboards/actions/
   shadow: true # Draws a drop shadow on icons (optional)
   hide_unavailable: false # Hide unavailable entities (optional)
   state_color: true # enable or disable HA colors for all entities
+  shadow: true # enable a drop shadow on entity icons to contrast with the background
+  darken_image: true # reduce brightness of the background image to constrast with entities
   entity_type: auto # auto, sensor, button (optional)
```

Comment:
`entity_type` should be part of the entity settings, can you please move it too?

Reply:
Cleaned in https://github.com/LesTR/homeassistant-minimalistic-area-card/pull/193